ev3/legoev3_motor.c

```diff
@@ -341,38 +323,29 @@ static irqreturn_t tacho_motor_isr(int irq, void *id)
 	return IRQ_HANDLED;
 }
 
-void legoev3_motor_update_output(struct legoev3_motor_data *ev3_tm)
+void legoev3_motor_set_power(struct legoev3_motor_data *ev3_tm, int duty_cycle)
```

Comment:
Instead of `power`, can we call this `legoev3_motor_set_duty_cycle` to keep naming consistent?